fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneOnlineIndexingTest.java

```diff
@@ -760,6 +780,46 @@ void luceneOnlineIndexingTestGroupingKeysBackgroundMerge(int groupingCount) thro
         assertTrue(newLength < oldLength);
     }
 
+    @SuppressWarnings("checkstyle:VariableDeclarationUsageDistance")
+    @ParameterizedTest
+    @BooleanSource
+    void luceneOnlineIndexingTestFilterOutRecords(boolean filterOut) throws IOException {
+        int groupingCount = 1;
+        final int groupedCount = 4 - groupingCount;
```

Comment:
These two could probably be inlined, although that might become moot if you use `LuceneIndexTestDataModel`

Reply:
√